### ISSUE TITLE
User icon misalignment

### ISSUE THREAD

**Issue by james-s-w-clark** [denysdovhan/inboxer#21]
![inboxer_imgdisplace](https://user-images.githubusercontent.com/12559216/34814959-831e24e2-f6ea-11e7-9869-58826a586988.png)

As you can see, the user icon actually dips below the blue bar.

Searched the code for "icon", saw this in _inboxer/app/renderer/browser.css_:

```
/* Google User */
.gb_hb.gb_5c.gb_Cg.gb_R {
  padding-left: 0;
}
.gb_b.gb_R,
.gb_bb.gbii {
  height: 26px;
  width: 26px;
  margin-top: 1px;
}
```

```margin-top: 1px; ``` seems to be too small to push the Google User icon down this far, though.

**Comment by realsrikar** [comment]
Has been fixed in v1.0.1 #8

**Comment by denysdovhan** [comment]
@IdiosApps should be fixed since `v1.0.1`. Feel free to reopen, if the problem was not solved after updating to `v1.0.1`.
